.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] [REQUIRED] Kindly add node_modules to the list of ignored files as instructed.